### DISCUSSION TITLE
Just to share an idea to bring back a abandon project (WX.NET) or multiplaform GUI for mono

### DISCUSSION THREAD

**Discussion by gnubsdguy** [mono/mono#21719] [Ideas]
Hi, Hello, I have been practising and learning Mono C# and using Monodevelop as my main IDE and develop platform for C# On Linux(I am learning C++ for knowledge). But what I cannot find is a standard GUI for Mono or Mono Develop (there is not a complete Win forms, and we only have GTK# (which I like it) I have been looking for something else and there was something called as WX.Net (check on source forge about this project). I know that all of you keep busy with maintenance of the project mono, monodevelop  and doing so many other things. I was thinking about because Wx Widgets is very well-used on Windows, Mac and Linux to bring alive again this project (I mean WX.NET it has not being updated since 2010) and may be to have another standard GUI library for multiplatform development with C# and Mono. There are so many people on different forums  asking what happened to WX.NET. I don't know but it would be nice to know what the developers of the mono project will think about this.

**Comment by lextm** [comment]
There are many popular options today to go to, so I don't think much attention will be put into WxWidgets,

https://docs.lextudio.com/blog/the-story-about-net-cross-platform-ui-frameworks-dd4a9433d0ea

**Comment by gnubsdguy** [comment]
Hi Lex Li

Thanks for your reply in any case I am not interested to use WX.Net but I
have seen many cool options as Avalonia, MAUI or UNO. In any case now I can
run a IDE (Finally) on linux and I am just keep to deep learn C# with Rider.

In any case thanks so much to answer me !

Regards

Manuel
